templates/CRM/common/civicrm.settings.php.template

```diff
@@ -243,27 +243,13 @@ if (!defined('CIVICRM_UF_BASEURL')) {
  * the near future - this opt-in setting is transitional.
  */
 if (!defined('CIVICRM_SMARTY_AUTOLOAD_PATH') && getenv('CIVICRM_SMARTY_AUTOLOAD_PATH') === FALSE) {
-  // enable by default for dev & demo sites for now - will soon enable by default for all new installs.
-  if (CIVICRM_UF === 'UnitTests' || strpos(CIVICRM_UF_BASEURL, 'localhost') !== FALSE || strpos(CIVICRM_UF_BASEURL, 'demo.civicrm.org') !== FALSE
-    || strpos(CIVICRM_UF_BASEURL, 'http://build') !== FALSE
-  ) {
-    if (is_dir($civicrm_root . '/packages')) {
-      define('CIVICRM_SMARTY_AUTOLOAD_PATH', $civicrm_root . '/packages/smarty5/Smarty.php');
-    }
-    elseif (is_dir($civicrm_root . '/../civicrm-packages')) {
-      define('CIVICRM_SMARTY_AUTOLOAD_PATH', $civicrm_root . '/../civicrm-packages/smarty5/Smarty.php');
-    }
-  }
-}
-
-// If not set above we should default the Smarty version to Smarty4 which is our stable version.
-// This will not work for all composer-based installs @todo.
-if (!defined('CIVICRM_SMARTY_AUTOLOAD_PATH') && getenv('CIVICRM_SMARTY_AUTOLOAD_PATH') === FALSE) {
+  // If not set we should default the Smarty version to Smarty4 which is our stable version.
```

Comment:
@eileenmcnaughton should this comment say Smarty 5 rather than 4?

Reply:
opps - will follow up